menu.js

```diff
@@ -0,0 +1,39 @@
+const lastWidth = document.querySelector('body').style.width;
+const lastHeight = document.querySelector('body').style.height;
+
+const closeMenu = function () {
+  document.getElementsByClassName('menu')[0].classList.add('hidden');
+  document.querySelector('body').style.width = lastWidth;
+  document.getElementsByClassName('headline')[0].classList.remove('hidden'); 
+  document.getElementsByClassName('Works')[0].classList.remove('hidden');
+  document.getElementsByClassName('About')[0].classList.remove('hidden'); 
+  document.getElementsByClassName('form')[0].classList.remove('hidden'); 
+};
+
+const openMenu = function () {
+  document.getElementsByClassName('menu')[0].classList.remove('hidden');
+  document.querySelector('body').style.width = '10vmax'; 
+  document.getElementsByClassName('headline')[0].classList.add('hidden'); 
+  document.getElementsByClassName('Works')[0].classList.add('hidden'); 
+  document.getElementsByClassName('About')[0].classList.add('hidden');
+  document.getElementsByClassName('form')[0].classList.add('hidden');
+};
+
+document
+  .getElementsByClassName('cancel')[0]
+  .addEventListener('click', closeMenu);
+document
+  .getElementsByClassName('menu-button-selector')[0]
+  .addEventListener('click', openMenu);
```

Comment:
- [ ] Good job so far, however Minimum mobile screen size is 375 px which should fit the screen size but as you can see below it doesn't follow the design layout as per Figma and you should have made the mobile menu responsive until it changes to desktop screen size.

https://user-images.githubusercontent.com/70269507/195045182-c4cefdfd-f41b-4789-b8b9-9dbd208974bd.mov